hydra-node/src/Hydra/Chain/Direct/Tx.hs

```diff
@@ -323,40 +336,24 @@ closeTx ::
   -- FIXME(SN): should also contain some Head identifier/address and stored Value (maybe the TxOut + Data?)
   (TxIn StandardCrypto, TxOut Era, Data Era) ->
   ValidatedTx Era
-closeTx snapshotNumber _utxo (headInput, headOutput, headDatumBefore) =
-  emptyTx
-    & withBody body
-    & withDatums datums
-    & withRedeemers redeemers
-    & withScripts scripts
+closeTx snapshotNumber _utxo (Api.fromLedgerTxIn -> headInput, Api.fromLedgerTxOut -> headOutputBefore, Api.fromLedgerData -> headDatumBefore) =
```

Comment:
This line is very long, putting them into the `where` would help readability I suppose.

Reply:
I know, but keep in mind that it's only temporary while working on converting to cardano-api, ultimately, none of those conversions are needed anymore.